practice/src/ts/views/customersView.ts

```diff
@@ -0,0 +1,36 @@
+//Import
+import { CUSTOMERS_API } from '../constants/constants';
+import { HttpService } from '../services/httpServices';
+import { genTable } from '../templates/tableTemplate';
+
+//Get HTML element
+const customersTable = document.querySelector('.customers-table-body')!;
```

Comment:
Should double-check here. DOM should not be queried at the moment the file has just been imported.